CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project (loosely) adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.0.2 - 2023-10-04
+
+### Added
+
+- Added new type called VCClaimsWithVCDataModel that can be used to create VCSDJWT
+
+### Removed
+
+- The CredentialStatus type as it was not properly defined based on the specfications [JWT and CWT Status List](https://datatracker.ietf.org/doc/html/draft-looker-oauth-jwt-cwt-status-list-01)
```

Comment:
Yes, you can use it. However, it's important to note that since we are choosing a different data model that follows the W3C style, we're moving away from current specifications.